openood/evaluators/base_evaluator.py

```diff
@@ -40,6 +40,8 @@ def eval_acc(self,
 
                 # forward
                 output = net(data)
+                if isinstance(output, tuple):
```

Comment:
when the output would be a tuple? could we prevent it?

Reply:
I have added my method to README.
RTS (random temperature scaling) is a method originally designed to estimate face uncertainty. It models data uncertainty through distributional estimation, utilizing learnable mean (feature) and variance (uncertainty). Interestingly, we discovered that the temperature scalar precisely corresponds to the scale of uncertainty noise implicitly added in the softmax function. RTS offers several advantages: 1) It adjusts the learning strength of both clean and noisy samples to enhance stability and accuracy during the training phase. 2) It provides a confidence score to detect input uncertainty. 3) In addition to the uncertainty score, the MSP score can be used to identify OOD data.